.github/workflows/external.yml

```diff
@@ -21,20 +23,47 @@ jobs:
           python-version: 3.9
           cache: "pip"
 
-      - name: Install dependencies
+      - name: Build sdist
         run: |
-          pip install --upgrade pip
-          pip install -r requirements.txt
-          pip install -r requirements-dev.txt
+          python -m pip install --upgrade pip
+          python -m pip install -U build pip setuptools
+          python -m pip install -U -r requirements.txt
+          python -m build --sdist
+
+      - name: Delete source directory
+        shell: bash
+        run: |
+          rm -rf $MODULE_NAME
+
+      - name: Uninstall all packages
+        run: |
+          python -m pip freeze > installed.txt
+          python -m pip uninstall -y -r installed.txt
+
```

Comment:
Since we're not running mypy or anything else from non-package requirements, this isn't really doing anything.

```suggestion
```